docs/reference-guides/rancher-webhook.md

```diff
@@ -17,6 +17,26 @@ Rancher-Webhook is crucial for Rancher to protect clusters against malicious att
 Rancher relies on the Rancher-Webhook as an integral part of its functionality. Without the webhook, Rancher would not be a complete product. 
 It provides essential protection for Rancher-managed clusters, preventing security vulnerabilities and ensuring the consistency and stability of the cluster.
 
+## What resources does the webhook validate?
+
+An in-progress list of the resources that the webhook validates can be found in the [webhook's repo](https://github.com/rancher/webhook/blob/release/v0.4/docs.md). Checks specific to one version can be found by viewing the `docs.md` file associated with a particular tag (note that webhook versions prior to `v0.3.6` won't have a version).
```

Comment:
```suggestion
An in-progress list of the resources that the webhook validates can be found in the [webhook's repo](https://github.com/rancher/webhook/blob/release/v0.4/docs.md). You can find checks specific to one version by viewing the `docs.md` file associated with a particular tag. Note that webhook versions prior to `v0.3.6` won't have a version tag.
```

Reply:
Is there a place we/users can find the Rancher version:Webhook version relationship?

Reply:
@btat You are right on the wording, the file is what is missing, not the version.

> Is there a place we/users can find the Rancher version:Webhook version relationship?

Unfortunately there isn't a good answer here. There's ~4 places that users can get this from:
1. The Rancher repo ([example](https://github.com/rancher/rancher/blob/d395864008ca5e19611ee080ee7f5338df46192d/package/Dockerfile#L61)), but this will require reading source code and I'll need to explain that it's the end part (i.e. the part after `+up`) that matters here.
2. The `rancher-webhook-min-version` and `rancher-webhook-version` settings, but this is inconsistent from version to version (newer versions will use version, older ones will use min-version) and I still need to explain the end part like in the first option
3. The webhook deployment's image tag. I can write a kubectl command for this, but an admin would need to execute it (hosted rancher customers would not be able to use this command).
4. The version of the application installed, which you can use the UI to browse (also requires the ability to view the local cluster and the ability to parse the version syntax like options 1 and 2).

Do you have a preference on which of these options (if any) I should use?

Reply:
I would lean towards option 1 since it only requires web access (and a bit of understanding) or maybe adding a table with the version associations that the Docs team maintains.

Reply:
Ok, I can implement that. 

For what it's worth, there's a planned improvement to our build system (https://github.com/rancher/rancher/pull/42000) - if/when that goes in, there should be a single, easy to read-file containing the version in a more understandable syntax, so it might get easier to read over time.

Reply:
I was implementing the guidance here, and figured it would be easier to just put a list together of the current versions for 2.7. I'll get the table going.

Reply:
I've added a table for all currently released 2.7 versions. I also added some information on Rancher versions and the compatibility with the webhook - please give that a look.